root/static/scripts/edit/forms.js

```diff
@@ -279,4 +279,4 @@ ko.bindingHandlers.withLabel = {
     }
 };
 
-module.exports = MB.forms;
+export default MB.forms;
```

Comment:
I'd prefer no default export here too, and adding named exports for `buildOptionsTree`, `linkTypeOptions`, etc., which is how this module was previously consumed.

`MB.forms` is a bogus namespace from before we had actual modules - given that this module holds several utility functions and doesn't have/need a primary export, it makes much more sense to use named exports.

Reply:
Do I just *remove* MB.forms completely then, leaving only separate functions?

Reply:
It's used in root/components/relationship-editor.tt by knockout so I wouldn't remove the reference, just the export for it.